parser_library/test/workspace/pathmask_test.cpp

```diff
@@ -78,6 +78,15 @@ TEST(pathmask, pass)
     EXPECT_TRUE(check_path("/path/**", "\\path\\test\\"));
     EXPECT_TRUE(check_path("/path/**", "\\path\\a\\test\\"));
     EXPECT_TRUE(check_path("/path/**", "\\path\\a\\b\\test\\"));
+
+    EXPECT_TRUE(check_path("file:///C%3A/path/**/", "file:///C%3A/path/a/test/"));
```

Comment:
Do you @slavek-kucera think we should allow URIs with `:` as well? E.g.
```
file:///C:/path/**/"
```
instead of 
```
file:///C%3A/path/**/"
```

I would say yes as it is more user friendly but it is not a standard URI.

Reply:
Is it actually an invalid URL?

More important question is that we should probably keep interpreting `C:/...` as path on windows even though it looks like an URL.